code/modules/antagonists/abductor/abductor_structures.dm

```diff
@@ -0,0 +1,123 @@
+
+// Operating Table / Beds / Lockers
+
+/obj/structure/bed/abductor
+	name = "resting contraption"
+	desc = "This looks similar to contraptions from Earth. Could aliens be stealing our technology?"
+	icon = 'icons/obj/abductor.dmi'
+	buildstacktype = /obj/item/stack/sheet/mineral/abductor
+	icon_state = "bed"
+
+/obj/structure/table_frame/abductor
+	name = "alien table frame"
+	desc = "A sturdy table frame made from alien alloy."
+	icon_state = "alien_frame"
+	framestack = /obj/item/stack/sheet/mineral/abductor
+	framestackamount = 1
+
+/obj/structure/table_frame/abductor/attackby(obj/item/I, mob/user, params)
+	if(I.tool_behaviour == TOOL_WRENCH)
+		to_chat(user, span_notice("You start disassembling [src]..."))
+		I.play_tool_sound(src)
+		if(I.use_tool(src, user, 30))
+			playsound(src, 'sound/items/deconstruct.ogg', 50, TRUE)
+			for(var/i in 0 to framestackamount)
+				new framestack(get_turf(src))
+			qdel(src)
+			return
+	if(istype(I, /obj/item/stack/sheet/mineral/abductor))
+		var/obj/item/stack/sheet/P = I
```

Comment:
no new single letter variables please, what the fuck is a `P`

Reply:
i later realize that this likely isn't a "new" thing, but should still be eradicated